docs/introduction.md

```diff
@@ -62,6 +62,20 @@ You can share class data between running VMs, which can reduce the startup time
 ### Native data operations
 If your Java application manipulates native data, consider writing your application to take advantage of methods in the Data Access Accelerator API.
 
+The following functions are provided:
+
+- Arithmetic, copying, and shifting operations for packed decimal data.
```

Comment:
'copying' should probably be 'comparison' -- though I noticed this text came directly from our API javadocs, which should be fixed too!   We might want to add validation operation to cover the checkPackedDecimal set of APIs.  I'd propose we go with:

> Arhithmetic, comparison, shifting and validation operations for packed decimal data.